### DISCUSSION TITLE
gettting information from standardUserDefaults and com.apple.configuration.managed

### DISCUSSION THREAD

**Discussion by trkas** [appcelerator/titanium_mobile#13215] [Q&A] [answered]
Hi,

Is it possible to read the NSUserDefaults com.apple.configuration.managed dictionary from a Titanium iOS app?

I want to push some configuration values from a MDM eg. SimpleMDM..

I have tried with Ti.App.iOS.createUserDefaults but how to get the standardUserDefaults?

**Comment by m1ga** [comment, marked as answer]
according to https://stackoverflow.com/a/54753783/5193915 the `Titanium.App.Properties` are stored in StandardUserDefaults. So it is the otherway round but you might be able to read the StandardUserDefaults with `Ti.App.Properties` too

**Reply by trkas** [reply]
Thank you!

I got it to work! 👍
